README.md

```diff
@@ -1,15 +1,19 @@
 # OpenTelemetry Semantic Conventions
 
-Welcome to the new repository!
+[![Checks](https://github.com/open-telemetry/semantic-conventions/workflows/Checks/badge.svg?branch=main)](https://github.com/open-telemetry/semantic-conventions/actions?query=workflow%3A%22Checks%22+branch%3Amain)
+[![GitHub tag (latest SemVer)](https://img.shields.io/github/tag/open-telemetry/semantic-conventions.svg?logo=opentelemetry&&color=f5a800&label=Latest%20release)](https://github.com/open-telemetry/semantic-conventions/releases/latest)
+[![Specification Version](https://img.shields.io/badge/OTel_specification_version-v1.26.0-blue?logo=opentelemetry&color=f5a800)](https://github.com/open-telemetry/opentelemetry-specification/releases/tag/v1.26.0)
 
-This is currently a direct copy/filter-branch of the Specification repository
-with only semantic conventions included.
+![OpenTelemetry Logo](https://opentelemetry.io/img/logos/opentelemetry-horizontal-color.png)
```

Comment:
This shows the log quite large (more than half of my screen).

Maybe something more subtle like in the collector docs will look better?

https://github.com/open-telemetry/opentelemetry-collector/blob/main/README.md?plain=1#L46

Reply:
I thought it would be a good idea to be the same as in the spec repo, but indeed that is much better. Changed!